client/src/helper/baseUrlHelper.ts

```diff
@@ -1,8 +1,9 @@
 export function getbaseURL() {
+  if (import.meta.env.VITE_BASE_USE_LIVE_API == "true")
```

Comment:
Warum musste das von der Logik umgedreht werden?

Reply:
Weil er sonst in production immer die baseURL abhängig vom host nimmt.

Wenn ich aber einen brach in dev auf einem live server testen will, muss ich forcen können, dass er die xccup.net domain für die API nimmt auch wenn die URL eine andere ist.